frontend/src/api/k8s/notebooks.ts

```diff
@@ -210,10 +237,19 @@ export const startNotebook = async (
   name: string,
   namespace: string,
   tolerationChanges: TolerationChanges,
+  dashboardConfig: DashboardConfig,
   enablePipelines?: boolean,
 ): Promise<NotebookKind> => {
-  const patches: Patch[] = [];
-  patches.push(startPatch);
+  const enableServiceMesh = featureFlagEnabled(
```

Comment:
I am not sure I understand. Are you suggesting creating a util function that getsAppContext, then calls featureFlagEnabled to parse the service mesh flag (therefore taking no inputs and returning this bool)?

Reply:
Yes, I'm talking about something like this: https://github.com/opendatahub-io/odh-dashboard/blob/main/frontend/src/pages/projects/projectSharing/utils.ts#L7 is not mandatory but it might be nice to have